docker/server/build.bash

```diff
@@ -75,6 +75,12 @@ function gzipJsonFiles {(
 
 function build {(
 
+  # Check if the jq processor is installed correctly
+  if ! which jq > /dev/null 2>&1; then
```

Comment:
Use `! command -v jq` instead -- see [this SO answer](https://stackoverflow.com/a/677212/7002313).